cpp/modules/deck.gl/core/src/lib/deck.h

```diff
@@ -101,7 +91,7 @@ class Deck::Props : public Component::Props {
   int width{100};   // Dummy value, ensure something is visible if user forgets to set window size
   int height{100};  // Dummy value, ensure something is visible if user forgets to set window size
 
-  std::optional<Deck::RenderingOptions> renderingOptions;
+  std::optional<lumagl::AnimationLoop::Options*> drawingOptions;
```

Comment:
Options object should typically not be pointers, they should be values!

Also If it is a pointer you can null it so no need for std::optional